README.md

```diff
@@ -12,6 +12,24 @@ Olam free and open source English Malayalam repository with Qt gui
 * Malayalam English
 * Malayalam Malayalam
 
+# Building from source
+
+### General Build Requirements
+* Qt 5.9 SDK or later
```

Comment:
I haven't actually tested with different versions of Qt, but ideally At 5.1 + should be enough